wagtailmodelchoosers/wagtail_hooks.py

```diff
@@ -17,10 +17,14 @@ def wagtailmodelchoosers_admin_css():
 
 @hooks.register('insert_editor_js')
 def wagtailmodelchoosers_admin_js():
-    return format_html(
```

Comment:
@loicteixeira I tried this and it doesn't work. Just breaking my HTML :/
I checked the documentation and I can't see anything wrong with your code... Confused...

I get this in my terminal
```
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
Traceback (most recent call last):
  File "/home/vagrant/kiwibank_public/kiwibank_public/jinja2.py", line 56, in _fail_with_undefined_error
    super()._fail_with_undefined_error(*args, **kwargs)
  File "/home/vagrant/pyenv/versions/kiwibank_public/lib/python3.6/site-packages/jinja2/runtime.py", line 609, in _fail_with_undefined_error
    raise self._undefined_exception(hint)
jinja2.exceptions.UndefinedError: 'wagtail.wagtailcore.blocks.struct_block.StructValue object' has no attribute 'link'
```

Works when I switch back to current code.